unity/Assets/Scripts/Content/ContentData.cs

```diff
@@ -1060,7 +1060,15 @@ public TileSideData(string name, Dictionary<string, string> content, string path
         // pixel per D2E square (inch) of image
         if (content.ContainsKey("pps"))
         {
-            float.TryParse(content["pps"], out pxPerSquare);
+            if (content["pps"].StartsWith("*"))
```

Comment:
Alright, now I feel stupit. This indeed did the trick on both Android and Windows.

Reply:
> Can you please submit the SoA updates in a separate pull request?

Will do